test/augmentation/test_augmentation_3d.py

```diff
@@ -759,3 +760,90 @@ def test_gradcheck(self, device):
         input = torch.rand((3, 3, 3)).to(device)  # 3 x 3 x 3
         input = utils.tensor_to_gradcheck_var(input)  # to var
         assert gradcheck(RandomRotation3D(degrees=(15.0, 15.0)), (input, ), raise_exception=True)
+
+
+class TestRandomEqualize3D:
+    def smoke_test(self, device, dtype):
+        f = RandomEqualize3D(0.5)
+        repr = "RandomEqualize3D(p=0.5, return_transform=False)"
+        assert str(f) == repr
+
+    def test_random_equalize(self, device, dtype):
+        f = RandomEqualize3D(p=1.0, return_transform=True)
+        f1 = RandomEqualize3D(p=0., return_transform=True)
+        f2 = RandomEqualize3D(p=1.)
+        f3 = RandomEqualize3D(p=0.)
+
+        bs, channels, depth, height, width = 1, 3, 6, 10, 10
+
+        inputs3d = self.build_input(channels, depth, height, width, device=device, dtype=dtype).squeeze(dim=0)
+        inputs3d.to(device).to(dtype)
+
+        row_expected = torch.tensor([
+            0.0000, 0.11764, 0.2353, 0.3529, 0.4706, 0.5882, 0.7059, 0.8235, 0.9412, 1.0000
+        ], device=device, dtype=dtype)
+        expected = self.build_input(channels, depth, height, width, bs=1, row=row_expected,
+                                    device=device, dtype=dtype)
+        expected = expected.to(device)
+
+        identity = torch.eye(4, device=device, dtype=dtype)
```

Comment:
I guess broadcasting makes the job here but to make it safe I'd use `eye_like`.
`identity = K.eye_like(4, expected)`